client/state/selectors/get-email-forwarding-mx-servers.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Retrieve a list of custom mx servers for a particular domian
+ *
+ * @param  {Object} state    Global state tree
+ * @param  {String} domainName domainName to request email forwards for
+ * @return {Object}          EmailForwards list
```

Comment:
We should probably use `{Object?}` or `{Object|null}` (I'm not sure which syntax is the correct one though - I've sent to much time in Php lately) to highlight the fact that this function can return `null` as well.

Reply:
looks like option 3, `{?Object}`